FFXIVVenues.Veni.csproj

```diff
@@ -40,9 +40,10 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.15.1" />
+    <PackageReference Include="moment.net" Version="1.3.2" />
     <PackageReference Include="NChronicle.Console" Version="1.1.1" />
     <PackageReference Include="System.Collections.Concurrent" Version="4.3.0" />
-    <PackageReference Include="FFXIVVenues.VenueModels" Version="6.7.0" />
+    <PackageReference Include="FFXIVVenues.VenueModels" Version="7.0.0" />

```

Comment:
Identified an issue in FFXIVVenues.VenueModels package wherein it did not compare OpenOverrides correctly in all instances. This has been fixed in version 7 and I'm just bringing the new version in here.